File: apps/api/src/lib/openrouter.ts
import OpenAI from 'openai'

export const openai = new OpenAI({
  apiKey: process.env.OPENROUTER_API_KEY!,
  baseURL: process.env.OPENROUTER_BASE_URL || 'https://openrouter.ai/api/v1',
  defaultHeaders: {
    'HTTP-Referer': process.env.FRONTEND_URL || 'http://localhost:3000',
    'X-Title': 'Geta-AI',
  },
})

File: apps/api/app/lib/openrouter.py
from openai import AsyncOpenAI

from app.config import settings


openai_client = AsyncOpenAI(
    api_key=settings.openrouter_api_key,
    base_url=settings.openrouter_base_url,
    default_headers={
        "HTTP-Referer": settings.frontend_url,
        "X-Title": "Geta-AI",
    },
)
